lib/produce/itunes_connect.rb

```diff
@@ -75,11 +77,16 @@ def initial_create
       wait_for_elements("input[ng-model='createAppDetails.newApp.name.value']").first.set @config[:app_name]
       wait_for_elements("input[ng-model='createAppDetails.versionString.value']").first.set @config[:version]
       wait_for_elements("input[ng-model='createAppDetails.newApp.vendorId.value']").first.set @config[:sku]
-      
-      wait_for_elements("option[value='#{@config[:bundle_identifier]}']").first.select_option
+
       all(:xpath, "//option[text()='#{@config[:primary_language]}']").first.select_option
+      wait_for_elements("option[value='#{@config[:bundle_identifier]}']").first.select_option
+
+      if @config[:bundle_identifier_suffix] != nil
```

Comment:
No it's correct. `@config[:bundle_identifier_suffix]` will prompt user for bundle suffix if it's not set.